### ISSUE TITLE
Doing high-pass filtering twice?

### ISSUE THREAD

**Issue by tien-tong** [poldracklab/ds003-post-fMRIPrep-analysis#8]
Hi,

I thought that high-pass filtering was already achieved by [including the DCT-basis regressors](https://github.com/poldracklab/ds003-post-fMRIPrep-analysis/blob/e300357c5714e56157e6aaf63b9b70beab3f8b42/workflows.py#L40) in the GLM model. However, you also have [an additional highpass filtering step](https://github.com/poldracklab/ds003-post-fMRIPrep-analysis/blob/e300357c5714e56157e6aaf63b9b70beab3f8b42/workflows.py#L49). Can you please explain why we need to do both?

Best,

Tien

**Comment by effigies** [comment]
I agree, this looks like it will duplicate regressors, unnecessarily. Assuming that uses a cosine regressor, rather than a polynomial, then there should be no effect, as any drift components will be split between the two, and discarded in either case.

Anyway, we should clean this up.
